src/puppetlabs/dujour/version_check.clj

```diff
@@ -118,7 +128,21 @@
   ([request-values update-server-url callback-fn]
     (validate-config! request-values update-server-url)
     (future
-      (let [server-response (try
+      (let [certname (:certname request-values)
+            request-values (if-not (nil? certname)
+                             (->
```

Comment:
it's odd to have a line break here, usually just put the first value on the line with the threading macro and then indent all the arguments to that